packages/core/src/config/shipping-method/shipping-calculator.ts

```diff
@@ -90,6 +87,11 @@ export interface ShippingCalculationResult {
     metadata?: Record<string, any>;
 }
 
+export type CalculateShippingFnResponse =
```

Comment:
I think a better name would be `CalculateShippingFnResult` - "response" is more associated with a network call.